contractcourt/chain_arbitrator.go

```diff
@@ -570,6 +546,90 @@ func (c *ChainArbitrator) Start() error {
 	return nil
 }
 
+// publishClosingTxs will load any stored cooperative or unilater closing
+// transactions and republish them. This helps ensure propagation of the
+// transactions in the event that prior publications failed.
+func (c *ChainArbitrator) publishClosingTxs(
+	channel *channeldb.OpenChannel) error {
+
+	// If the channel has had its unilateral close broadcasted already,
+	// republish it in case it didn't propagate.
+	if channel.HasChanStatus(channeldb.ChanStatusCommitBroadcasted) {
```

Comment:
Perhaps these two should be mutually exclusive? Or even maybe we should prefer the _coop_ close as it doesn't lead to coins being time locked.

Reply:
that's definitely possible, favoring the coop close sounds good to me since the force close would probably win out due to the higher fee rate. this is preexisting tho, and fixing it will sprawl a bit so maybe we do it in a followup? as is we fix the issue of possibly overwriting a one transaction with another

Reply:
discussed offline, leave for follow up